e2e/connection_test.go

```diff
@@ -2,20 +2,76 @@ package e2e
 
 import (
 	"flag"
+	"github.com/cihub/seelog"
 	"github.com/mysterium/node/tequilapi/client"
 	"github.com/stretchr/testify/assert"
 	"testing"
+	"time"
 )
 
 var host = flag.String("tequila.host", "localhost", "Specify tequila host for e2e tests")
 var port = flag.Int("tequila.port", 4050, "Specify tequila port for e2e tests")
 
-func TestClientStatusIsNotConnectedOnStartup(t *testing.T) {
+func TestClientConnectsToNode(t *testing.T) {
 	//we cannot move tequilApi as var outside - host and port are not initialized yet :(
 	//something related to how "go test" calls flag.Parse()
 	tequilApi := client.NewClient(*host, *port)
 
 	status, err := tequilApi.Status()
 	assert.NoError(t, err)
 	assert.Equal(t, "NotConnected", status.Status)
+
+	identities, err := tequilApi.GetIdentities()
+	assert.NoError(t, err)
+
+	var identity client.IdentityDTO
+	if len(identities) < 1 {
+		identity, err = tequilApi.NewIdentity("")
```

Comment:
It's 2nd green path. Should not we have 2 separate tests for that?

Reply:
It's not green path. It's ensurement that we have available identity to use for connection - test cannot proceed without that.